Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient.m

```diff
@@ -213,9 +213,11 @@ - (BUYRequestOperation *)requestForURL:(NSURL *)url method:(NSString *)method ob
 	request.HTTPMethod = method;
 	
 	BUYRequestOperation *operation = [[BUYRequestOperation alloc] initWithSession:self.session request:request payload:object completion:^(NSDictionary *json, NSHTTPURLResponse *response, NSError *error) {
-		[self.callbackQueue addOperationWithBlock:^{
-			completionHandler(json, response, error);
-		}];
+		[self.callbackQueue addOperation:[NSBlockOperation blockOperationWithBlock:^{
```

Comment:
We can likely switch back to using `addOperationWithBlock:`. Since the checkout operation is no longer performed on the main thread, it shouldn't deadlock.